src/main/java/crud/controller/driver/GetAllDriversController.java

```diff
@@ -0,0 +1,25 @@
+package crud.controller.driver;
+
+import crud.lib.Injector;
+import crud.model.Driver;
+import crud.service.abstraction.DriverService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetAllDriversController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("crud");
+    private static final DriverService driverService
```

Comment:
it's not a CONSTANT, use common camelCase

Reply:
maybe I've misunderstood, but why is it not?